### DISCUSSION TITLE
Feasycom iBeacon showing in ESPresence but not being passed to HA

### DISCUSSION THREAD

**Discussion by nic-nicol** [ESPresense/ESPresense#960] [General]
Bought a few Feasycom FSC_BP103B to go on keyrings.  They come with their MAC address printed on them so are easy to identify in the ESPresence web terminal - highlighted pink

![image](https://github.com/ESPresense/ESPresense/assets/98160640/3fe24f06-da7a-4594-8ff6-67eb988f800e)

Have copied the yaml from the homepage, just changing "ibeacon: xxx" to the one showing in the web terminal.

![image](https://github.com/ESPresense/ESPresense/assets/98160640/c8d4af30-4245-4671-90d3-538b90021789)

But nothing shows up in HA.  I can see and set the sensors in Dev tools and on an entities card my phone shows it's area, but the beacons are not available/not home. In ESPresence I tried marking one as "known", but that didn't help.
![image](https://github.com/ESPresense/ESPresense/assets/98160640/8ef11397-5d0d-453b-9c26-6e191b4b3717)

I'm obviously missing something simple (that's usually the case) But why does the yaml work for my phone, but not the beacons? 

I tried using ESPHome and that passes their data to HA, but  ESPresence just doesn't.  I'd rather use Presence as it's more configurable for room tracking (being able to set range etc).  Any advice would be gratefully appreciated, cheers!

**Comment by DTTerastar** [comment]
Those timeouts are pretty low:

timeout: 10
away_timeout: 120

is the suggested.

**Reply by nic-nicol** [reply]
Thanks. I tried changing back to those settings, unfortunately it hasn't made any difference. 
I changed to the lower values after watching Alan Byrne's tutorial  on the quick start page (Home Automation Guy) My iPhone has the same timeout settings and it's fine.

**Comment by nic-nicol** [comment]
I've installed mqtt explorer to see if messages are being passed to HA. They are!

![image](https://github.com/ESPresense/ESPresense/assets/98160640/0e5c5e45-12b7-4742-8cbd-5b14d8bba70c)

This is the contents of that packet:

{"id":"known:dc0d301432ec","idType":210,"rssi@1m":-75,"rssi":-34,"raw":0.07,"distance":1.82,"speed":-0.10,"mac":"dc0d301432ec","close":true,"interval":1868,"mV":2987,"temp":128.0}

It's showing up every time I move it in/out of range:

![image](https://github.com/ESPresense/ESPresense/assets/98160640/0870a1ea-1953-4722-99a6-83fcca5aaff9)

Just looking in entities menu and reckon this must be where the problem lies, all the relevant entities I'm trying to track have this warning I've never seen before, when hovered over it says 'restored'

![image](https://github.com/ESPresense/ESPresense/assets/98160640/3cee01fe-9995-44b0-ae33-66ff3e009075)


When I click on the warning I get "This entity is no longer being provided by the ibeacon integration. If the entity is no longer in use, delete it in settings.". So this will the issue, but I don't have a clue what to do about it!! 😂🤣

![image](https://github.com/ESPresense/ESPresense/assets/98160640/1eefda9d-6976-4bf8-afde-5006db874232)

**Reply by DTTerastar** [reply]
You are confusing the iBeacon support native to home assistant (via bluetooth adapter or bluetooth proxy) with espresense.  If it was from espresense it would be from mqtt.  Did you do this: https://espresense.com/home_assistant#beacons-configurationyaml

**Reply by nic-nicol** [reply]
I certainly am confused!
Yeah, I've added that to my config yaml. 
The only changes I've made are the timeout values and the state topic to 'espressence/rooms' instead of 'espressence/ *MAC ADDRESS* ' in the tutorial.. I changed it because I've seen others use it, but neither worked.
This is my yaml, it works perfectly for my iphone, but not the 2 key fobs.

![image](https://github.com/ESPresense/ESPresense/assets/98160640/bbdb86ff-4cf6-4de3-9305-d2db92ad7d4e)
